onc/+onc/DataProductFile.m

```diff
@@ -101,31 +99,43 @@
                         this.fileSize = strlength(response.Body.Data);
                     end
                     %save_as_file doesn't work properly! Use urlwrite instead
-                    [~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
-                    %saveResult = util.save_as_file(response, outPath, filename, overwrite);
+                    %[~, saveResult] = urlwrite(uri.EncodedURI,fullfile(outPath, filename));
+                    % neither urlwrite nor save_as_file works properly, use websave
```

Comment:
Can you update this comment to make it in line with the comment on line 101? 

101 says to use urlwrite, then this says not to use urlwrite

Reply:
Sure! I forgot the commented lines